sceptre/scipool/templates/sc-portfolio-lambda.yaml

```diff
@@ -0,0 +1,34 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Lambda Portfolio for Service Catalog
+Parameters:
+  PortfolioProvider:
+    Type: String
+    Description: Provider Name
+    Default: Sage Bionetworks
+  PorfolioName:
+    Type: String
+    Description: Portfolio Name
+    Default: Lambda Portfolio
+  PorfolioDescription:
+    Type: String
+    Description: Portfolio Description
+    Default: Portfolio of Lambda products
+Resources:
+  SCLambdaPortfolio:
+    Type: AWS::ServiceCatalog::Portfolio
+    Properties:
+      ProviderName: !Ref 'PortfolioProvider'
+      Description: !Ref 'PorfolioDescription'
+      DisplayName: !Ref 'PorfolioName'
+  LinkEndusersRole:
+    Type: AWS::ServiceCatalog::PortfolioPrincipalAssociation
+    Properties:
+      PrincipalARN: !ImportValue
+        'Fn::Sub': '${AWS::Region}-sc-enduser-iam-ServiceCatalogEndusers-RoleArn'
```

Comment:
This is the association that limits the product to Sage Staff (the Endusers role) so that ExternalEndusers cannot access the product, correct?

Reply:
yes, correct.